src/HostnetExperimental/ruleset.xml

```diff
@@ -2,22 +2,4 @@
 <ruleset name="HostnetExperimental">
     <description>The future Hostnet coding standard.</description>
```

Comment:
what is the motivation for keeping this ruleset?

Reply:
As discussed, the file has to be kept for future reference.